### PR TITLE
add wolfi-based nginx image

### PR DESCRIPTION
**Pull request by imjasonh**
Signed-off-by: Jason Hall <jason@chainguard.dev>

### CODE REVIEW

File: images/nginx/configs/latest-glibc.apko.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+contents:
+  repositories:
+    - https://packages.wolfi.dev/os
+  packages:
+    - nginx
```
Comment (dlorenc) [inline images/nginx/configs/latest-glibc.apko.yaml:5]: We don't have nginx yet in wolfi, i think? https://github.com/wolfi-dev/os

[the remark this answers is not in the document]
Comment (rawlingsj) [reply images/nginx/configs/latest-glibc.apko.yaml:5]: ah I submitted my comment at the same time - yeah, fwiw this is the work in progress melange package https://github.com/rawlingsj/temp-melange-nginx/blob/main/200-nginx.yaml

**Review by jdolitsky — CHANGES_REQUESTED** [review]
variant needs to be added to image.yaml

**Comment by imjasonh** [comment]
Ah for some reason I thought I saw nginx in wolfi. Closing until we have that.

**Pull request closed without merging**